apps/web/pages/settings/my-account/conferencing.tsx

```diff
@@ -46,7 +46,7 @@ const AddConferencingButton = () => {
 
   return (
     <>
-      <Button color="secondary" StartIcon={Plus} href="/apps/categories/video">
+      <Button color="secondary" StartIcon={Plus} href="/apps/categories/conferencing">
```

Comment:
@gitstart Can you confirm why it's not working on cal.com but working on app.cal.com as mentioned [here](https://github.com/calcom/cal.com/issues/11196#issuecomment-1710256296)

Reply:
@gitstart-calcom i don't see any problem on app.cal.com 
<img width="1440" alt="Screenshot 2023-09-22 at 5 54 57 PM" src="https://github.com/calcom/cal.com/assets/53316345/1463ee2f-f9ae-4269-93f7-7087bc0bda32">

Reply:
Hi @Udit-takkar 
Yeah, there's no issue on app.cal.com but on cal.com
https://www.loom.com/share/596dd405fa2e4b17a1c0d90502ecc71d

On cal.com too, works fine after a reload when it redirects to `/conferencing`

Reply:
We were able to reproduce the same issue locally and pushed this PR as a fix.
Please check out this demo
https://www.loom.com/share/3dbe7344a4e34ac296e5549da03e1b15